app/main.py

```diff
@@ -95,16 +104,68 @@ def _decrypt(self, encrypted_jwt, tx_id):
             raise RetryableError()
 
     def run(self):
+        logger.debug("Starting consumer")
         self.consumer.run()
 
     def stop(self):
+        logger.debug("Stopping consumer ")
         self.consumer.stop()
 
 
+class HealthCheck(tornado.web.RequestHandler):
+
+    @staticmethod
+    def rabbit_health():
+        http_client = HTTPClient()
+        resp = http_client.fetch(settings.RABBIT_HEALTHCHECK_URL, auth_username=settings.SEFT_RABBITMQ_MONITORING_USER,
+                                 auth_password=settings.SEFT_RABBITMQ_MONITORING_PASS)
+        body = resp.body.decode('utf8')
+        body = json.loads(body)
+        if body.get('status') == "ok":
+            health = body.get('status')
+        else:
+            health = "failed"
+        return health
+
+    @staticmethod
+    def ftp_health():
+        ftp = SDXFTP(logger, settings.FTP_HOST, settings.FTP_USER, settings.FTP_PASS, settings.FTP_PORT)
+        try:
+            ftp.get_connection()
+            health = "ok"
+        except Error as e:
+            logger.debug("FTP error raised" + e)
+            health = "failed"
+        return health
```

Comment:
Can just return as soon as you make the decision rather than assign:
```
try:
   ftp.get_connection()
   return "ok"
except Error as e:
  logger.info( ... )
  return "failed"
```